src/rgw/rgw_op.h

```diff
@@ -181,6 +182,7 @@ class RGWOp : public DoutPrefixProvider {
   RGWQuotaInfo user_quota;
   int op_ret;
   int do_aws4_auth_completion();
+  jspan multipart_trace;
```

Comment:
why is this part of `RGWOp` ?
this is needed only for multipart init/complete and for put?

Reply:
see https://github.com/ceph/ceph/pull/43372/#discussion_r721488093

Reply:
the comment from @cbodley was to make sure that the name is not confusing, and it is clear it is specific to multipart.
but would still only add that to the ops that needs it: "multipart init", "multipart complete" and "put"